### ISSUE TITLE
repo pruner

### ISSUE THREAD

**Issue by kfarkasHU** [wraithlight/wraithlight-mono#820]
write a tool that can prune the repo into a subset of it
since this monorepo is configured to use yarn workspaces, that can be used (lerna list can help a lot)
once it is done create a new ticket that is responsible for the partial yarnlock file

question 1: how .gitignores should be handled?
answer 1: nohow, specify the files/folders that should be copied
